aw-transform/src/flood.rs

```diff
@@ -126,4 +132,37 @@ mod tests {
         assert_eq!(&res[0], &e1_expected);
         assert_eq!(&res[1], &e2_expected);
     }
+
+    #[test]
+    fn test_flood_same_timestamp() {
+        let e1 = Event {
+            id: None,
+            timestamp: DateTime::from_str("2020-06-16T03:29:11.334000+00:00").unwrap(),
+            duration: Duration::microseconds(305402078),
```

Comment:
It's a bit hard to read these tests when you involve millisecond-precision. Would be nicer with round datetimes.

Reply:
Sorry, my bad. Was copied straight from a real broken example.
Fixed.